diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6ddaa51..93b1e92 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -69,9 +69,9 @@ write_basic_package_version_file(
     VERSION ${LLAMA_INSTALL_VERSION}
     COMPATIBILITY SameMajorVersion)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/LlamaConfig.cmake
-              ${CMAKE_CURRENT_BINARY_DIR}/LlamaConfigVersion.cmake
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Llama)
+# install(FILES ${CMAKE_CURRENT_BINARY_DIR}/LlamaConfig.cmake
+#               ${CMAKE_CURRENT_BINARY_DIR}/LlamaConfigVersion.cmake
+#         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Llama)
 
-set_target_properties(llama PROPERTIES PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/llama.h)
-install(TARGETS llama LIBRARY PUBLIC_HEADER)
\ No newline at end of file
+# set_target_properties(llama PROPERTIES PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/llama.h)
+# install(TARGETS llama LIBRARY PUBLIC_HEADER)
\ No newline at end of file